src/stream.ts

```diff
@@ -56,6 +59,9 @@ export class WebRTCStream implements Stream {
   closed: boolean = false;
   closeCb?: (stream: WebRTCStream) => void | undefined
 
+  // read state
+  messageState: MessageState = new MessageState();
```

Comment:
Why is this created but never used?

Reply:
artifact of a previous implementation. Removed now.